### PR TITLE
Andjd/lumberjack

### PR DESCRIPTION
**Pull request by andjd**
Adds Timber library for logging.

### How was this tested?
- [ ] I tested this manually on `debug`
- [ ] I tested this manually on `staging` or `release`
- [ ] I added automated tests
- [ ] Existing automated tests cover some of my changes
- [ ] Existing automated tests cover all of my changes
- [ ] I checked that `@Previews` still work in Android Studio

## Issues addressed
* https://github.com/neevaco/neeva-android/issues/994

### CODE REVIEW

**Review by dan-alcantara-neeva — COMMENTED** [review]
Guessing we're planning on converting logging code as we go?

**Comment by andjd** [comment]
I went and changed what logging calls there were at the time. There's no harm in having calls to Timber and Log for a while during a transition period.

**Comment by dan-alcantara-neeva** [comment]
> I went and changed what logging calls there were at the time. There's no harm in having calls to Timber and Log for a while during a transition period.

There are definitely a ton more existing logs than the ones you found.  I'm fine with converting them as we go, but I'm not sure why you focused on just those three files.

File: app/src/androidTest/java/com/neeva/app/ComposeTestUtils.kt [hunk 1]
```diff
@@ -20,6 +19,7 @@ import androidx.lifecycle.Lifecycle
 import androidx.test.platform.app.InstrumentationRegistry
 import java.util.concurrent.TimeUnit
 import org.junit.rules.TestRule
+import timber.log.Timber
 
 private const val TAG = "ComposeTestUtils"
```
Comment (dan-alcantara-neeva) [inline app/src/androidTest/java/com/neeva/app/ComposeTestUtils.kt:24]: Can you get rid of this TAG now?

File: app/src/main/java/com/neeva/app/NeevaBrowser.kt [hunk 2]
```diff
@@ -29,6 +30,10 @@ open class NeevaBrowser : Application() {
             )
         }
 
+        if (BuildConfig.DEBUG) {
+            Timber.plant(Timber.DebugTree())
```
Comment (dan-alcantara-neeva) [inline app/src/main/java/com/neeva/app/NeevaBrowser.kt:34]: Man, they like their puns...

**Review by dan-alcantara-neeva — APPROVED** [review]

**Pull request merged**